### ISSUE TITLE
Show name of column in tooltip?

### ISSUE THREAD

**Issue by iskandr** [hammerlab/cycledash#733]
When I scroll past the first page of variants, it's hard to tell which column all the numeric values correspond to (e.g. `[1, 1]	[42, 42]	43	0	[0, 0]	0	0`). One way to clarify this is to add the column name to the tooltip that pops up on hover.

**Comment by danvk** [comment]
We could also stick the header, like how it works on a spreadsheet.

**Comment by danvk** [comment]
@jaclynperrone

**Comment by hammer** [comment]
+1 sticky header. The first thing I do with every Google Spreadsheet is to pull that thing on the left down so that the first row becomes a sticky header row.

**Comment by jaclynperrone** [comment]
Yeah it would be great to have a sticky header. Will need to do some research to see how we can make that play nicely with a table.

**Comment by jaclynperrone** [comment]
cf. #800
